src/SharedWeb/Utilities/ServiceCollectionExtensions.cs

```diff
@@ -176,6 +177,11 @@ public static void AddDefaultServices(this IServiceCollection services, GlobalSe
         services.AddSingleton<IStripeSyncService, StripeSyncService>();
         services.AddSingleton<IMailService, HandlebarsMailService>();
         services.AddSingleton<ILicensingService, LicensingService>();
+        services.AddSingleton(provider =>
```

Comment:
Should we be registering the `ILookupClient` interface here? Something like this:

```
services.AddSingleton<ILookupClient>((serviceProvider) =>
{
       var options = new LookupClientOptions { Timeout = TimeSpan.FromSeconds(15) };
       return new LookupClient(options);
});
```

Reply:
Yeah, I skipped that contract. Thanks.